### PR TITLE
[zh] sync content/en/docs/tutorials/security/seccomp.md 

### PR DESCRIPTION
**Pull request by howieyuen**
close https://github.com/kubernetes/website/issues/31869

### CODE REVIEW

File: content/zh/docs/tutorials/security/seccomp.md [hunk 1]
```diff
@@ -0,0 +1,787 @@
+---
+reviewers:
+- hasheddan
+- pjbgf
+- saschagrunert
+title: 使用 seccomp 限制容器的系统调用
```
Comment (jihoon-seo) [inline content/zh/docs/tutorials/security/seccomp.md:6]: How about removing "reviewers" lines?
It seems that they are EN reviewers. 😊

```suggestion
---
title: 使用 seccomp 限制容器的系统调用
```

(For anyone's information:
EN file: https://github.com/kubernetes/website/blob/main/content/en/docs/tutorials/security/seccomp.md?plain=1 )

File: content/zh/docs/tutorials/security/seccomp.md [hunk 2]
```diff
@@ -0,0 +1,787 @@
...
+* Learn how to apply a seccomp profile to a container
+* Observe auditing of syscalls made by a container process
+* Observe behavior when a missing profile is specified
+* Observe a violation of a seccomp profile
+* Learn how to create fine-grained seccomp profiles
+* Learn how to apply a container runtime default seccomp profile
+-->
+* 了解如何在节点上加载 seccomp 配置文件
+* 了解如何将 seccomp 配置文件应用到容器中
+* 观察容器进程执对系统调用的审计
```
Comment (mengjiao-liu) [inline content/zh/docs/tutorials/security/seccomp.md:60]: “容器进程执”？

File: content/zh/docs/tutorials/security/seccomp.md [hunk 3]
```diff
@@ -0,0 +1,787 @@
...
+
+<!-- The Pod should be showing as having started successfully: -->
+此 Pod 应该显示为成功启动：
+```
+NAME        READY   STATUS    RESTARTS   AGE
+default-pod 1/1     Running   0          20s
+```
+
+<!-- Finally, now that you saw that work OK, clean up: -->
+最后，既然你看在可以运行，请清理：
```
Comment (mengjiao-liu) [inline content/zh/docs/tutorials/security/seccomp.md:770]:
```suggestion
最后，既然你看到可以运行，请清理：
```

File: content/zh/docs/tutorials/security/seccomp.md (hunk 2 above)
Comment (howieyuen) [reply content/zh/docs/tutorials/security/seccomp.md:60]: oops...typo

File: content/zh/docs/tutorials/security/seccomp.md [hunk 4]
```diff
@@ -0,0 +1,783 @@
...
+为简单起见，[kind](https://kind.sigs.k8s.io/) 可用来创建加载了 seccomp 配置文件的单节点集群。
+Kind 在 Docker 中运行 Kubernetes，因此集群的每个节点都是一个容器。
+这允许将文件挂载到每个容器的文件系统中，类似于将文件加载到节点上。
+
+{{< codenew file="pods/security/seccomp/kind.yaml" >}}
+
+<!-- 
+Download that example kind configuration, and save it to a file named `kind.yaml`:
+-->
+下载该示例类型配置，并将其保存到名为 `kind.yaml` 的文件中：
```
Comment (tengqm) [inline content/zh/docs/tutorials/security/seccomp.md:161]:
```suggestion
下载该示例 kind 配置，并将其保存到名为 `kind.yaml` 的文件中：
```

File: content/zh/docs/tutorials/security/seccomp.md [hunk 5]
```diff
@@ -0,0 +1,783 @@
...
+<!-- 
+Enabling the feature will neither change the Kubernetes
+`securityContext.seccompProfile` API field nor add the deprecated annotations of
+the workload. This provides users the possibility to rollback anytime without
+actually changing the workload configuration. Tools like
+[`crictl inspect`](https://github.com/kubernetes-sigs/cri-tools) can be used to
+verify which seccomp profile is being used by a container.
+-->
+启用该功能既不会更改 Kubernetes `securityContext.seccompProfile` API 字段，
+也不会新增已弃用的工作负载注解。
```
Comment (tengqm) [inline content/zh/docs/tutorials/security/seccomp.md:288]:
```suggestion
也不会添加已弃用的工作负载注解。
```

File: content/zh/docs/tutorials/security/seccomp.md [hunk 6]
```diff
@@ -0,0 +1,783 @@
...
+profile. To mitigate such a failure, you can:
+
+- Run the workload explicitly as `Unconfined`.
+- Disable the `SeccompDefault` feature for the nodes. Also making sure that
+  workloads get scheduled on nodes where the feature is disabled.
+- Create a custom seccomp profile for the workload.
+-->
+与其他工作负载相比，某些工作负载可能需要更少的系统调用限制。
+这意味着即使使用 `RuntimeDefault` 配置文件，它们也可能在运行时失败。
+要减轻此类故障，你可以：
```
Comment (tengqm) [inline content/zh/docs/tutorials/security/seccomp.md:306]:
```suggestion
要应对此类故障，你可以：
```

File: content/zh/docs/tutorials/security/seccomp.md [hunk 7]
```diff
@@ -0,0 +1,783 @@
...
+<!-- 
+If you were introducing this feature into production-like cluster, the Kubernetes project
+recommends that you enable this feature gate on a subset of your nodes and then
+test workload execution before rolling the change out cluster-wide.
+
+More detailed information about a possible upgrade and downgrade strategy can be
+found in the [related Kubernetes Enhancement Proposal (KEP)](https://github.com/kubernetes/enhancements/tree/a70cc18/keps/sig-node/2413-seccomp-by-default#upgrade--downgrade-strategy).
+-->
+如果你将此功能引入到类似生产的集群中，
+Kubernetes 项目建议你在节点的子集上启用此特性门控，
```
Comment (tengqm) [inline content/zh/docs/tutorials/security/seccomp.md:321]:
```suggestion
Kubernetes 项目建议你在部分节点上启用此特性门控，
```

File: content/zh/docs/tutorials/security/seccomp.md [hunk 8]
```diff
@@ -0,0 +1,783 @@
...
+## Create a Pod with a seccomp profile for syscall auditing
+
+To start off, apply the `audit.json` profile, which will log all syscalls of the
+process, to a new Pod.
+
+Here's a manifest for that Pod:
+-->
+## 使用 seccomp 配置文件创建 Pod 以进行系统调用审计 {#create-a-pod-with-a-seccomp-profile-for-syscall-auditing}
+
+首先，将 `audit.json` 配置文件应用到新的 Pod 中，该配置文件将记录进程的所有系统调用。
```
Comment (tengqm) [inline content/zh/docs/tutorials/security/seccomp.md:411]:
```suggestion
首先，将 `audit.json` 配置文件应用到新的 Pod 上，该配置文件将记录进程的所有系统调用。
```

File: content/zh/docs/tutorials/security/seccomp.md [hunk 9]
```diff
@@ -0,0 +1,783 @@
...
+
+{{< note >}}
+<!-- 
+The functional support for the already deprecated seccomp annotations
+`seccomp.security.alpha.kubernetes.io/pod` (for the whole pod) and
+`container.seccomp.security.alpha.kubernetes.io/[name]` (for a single container)
+is going to be removed with the release of Kubernetes v1.25. Please always use
+the native API fields in favor of the annotations.
+-->
+对已弃用的 seccomp 注解 `seccomp.security.alpha.kubernetes.io/pod`（针对整个 Pod）和
```
Comment (tengqm) [inline content/zh/docs/tutorials/security/seccomp.md:425]:
```suggestion
已弃用的 seccomp 注解 `seccomp.security.alpha.kubernetes.io/pod`（针对整个 Pod）和
```

File: content/zh/docs/tutorials/security/seccomp.md [hunk 10]
```diff
@@ -0,0 +1,783 @@
...
+`container.seccomp.security.alpha.kubernetes.io/[name]`（针对单个容器）
+将随着 Kubernetes v1.25 的发布而被删除。
+请始终使用原生 API 字段来支持注解。
```
Comment (tengqm) [inline content/zh/docs/tutorials/security/seccomp.md:428]:
```suggestion
请在可能的情况下使用原生 API 字段而不是注解。
```

File: content/zh/docs/tutorials/security/seccomp.md [hunk 11]
```diff
@@ -0,0 +1,783 @@
...
+NAME        TYPE       CLUSTER-IP      EXTERNAL-IP   PORT(S)          AGE
+audit-pod   NodePort   10.111.36.142   <none>        5678:32373/TCP   72s
+```
+
+<!-- 
+Now you can use `curl` to access that endpoint from inside the kind control plane container,
+at the port exposed by this Service. Use `docker exec` to run the `curl` command within the
+container belonging to that control plane container:
+-->
+现在，你可以使用 `curl` 在该服务公开的端口从 kind 控制平面容器内部访问该端点。
```
Comment (tengqm) [inline content/zh/docs/tutorials/security/seccomp.md:485]:
```suggestion
现在，你可以使用 `curl` 从 kind 控制平面容器内部访问该端点，位于该服务所公开的端口上。
```

File: content/zh/docs/tutorials/security/seccomp.md [hunk 12]
```diff
@@ -0,0 +1,783 @@
...
+## Create Pod with seccomp profile that causes violation
+
+For demonstration, apply a profile to the Pod that does not allow for any
+syscalls.
+
+The manifest for this demonstration is:
+-->
+## 使用导致违规的 seccomp 配置文件创建 Pod {#create-pod-with-seccomp-profile-that-causes-violation}
+
+为了演示，将配置文件应用于不允许任何系统调用的 Pod。
```
Comment (tengqm) [inline content/zh/docs/tutorials/security/seccomp.md:560]:
```suggestion
出于演示目的，将配置文件应用于不允许任何系统调用的 Pod 上。
```

File: content/zh/docs/tutorials/security/seccomp.md [hunk 13]
```diff
@@ -0,0 +1,783 @@
...
+As seen in the previous example, the `http-echo` process requires quite a few
+syscalls. Here seccomp has been instructed to error on any syscall by setting
+`"defaultAction": "SCMP_ACT_ERRNO"`. This is extremely secure, but removes the
+ability to do anything meaningful. What you really want is to give workloads
+only the privileges they need.
+
+Clean up that Pod before moving to the next section:
+-->
+如上例所示，`http-echo` 进程需要相当多的系统调用。
+这里 seccomp 已被指示通过设置 `"defaultAction": "SCMP_ACT_ERRNO"` 在任何系统调用上出错。
```
Comment (tengqm) [inline content/zh/docs/tutorials/security/seccomp.md:599]:
```suggestion
这里 seccomp 已通过设置 `"defaultAction": "SCMP_ACT_ERRNO"` 被指示为 在发生任何系统调用时报错。
```

File: content/zh/docs/tutorials/security/seccomp.md [hunk 14]
```diff
@@ -0,0 +1,783 @@
...
+这是非常安全的，但消除了做任何有意义的事情的能力。
+你真正想要的是只给工作负载他们需要的权限。
```
Comment (tengqm) [inline content/zh/docs/tutorials/security/seccomp.md:601]:
```suggestion
你真正想要的是只给工作负载它们所需要的权限。
```

File: content/zh/docs/tutorials/security/seccomp.md [hunk 15]
```diff
@@ -0,0 +1,783 @@
...
+sent to `syslog`.
+
+The manifest for this example is:
+-->
+## 使用只允许必要的系统调用的 seccomp 配置文件创建 Pod {#create-pod-with-seccomp-profile-that-only-allows-necessary-syscalls}
+
+如果你看一看 `fine-grained.json` 配置文件，
+你会注意到第一个示例的 syslog 中看到的一些系统调用，
+其中配置文件设置为 `"defaultAction": "SCMP_ACT_LOG"`。
+现在配置文件正在设置 `"defaultAction": "SCMP_ACT_ERRNO"`,
```
Comment (tengqm) [inline content/zh/docs/tutorials/security/seccomp.md:626]:
```suggestion
现在的配置文件设置 `"defaultAction": "SCMP_ACT_ERRNO"`,
```

File: content/zh/docs/tutorials/security/seccomp.md [hunk 16]
```diff
@@ -0,0 +1,783 @@
...
+-->
+打开一个新的终端窗口并使用 `tail` 来监视提到来自 `http-echo` 的调用的日志条目：
+
+```shell
+# 你计算机上的日志路径可能与 “/var/log/syslog” 不同
+tail -f /var/log/syslog | grep 'http-echo'
+```
+
+<!-- Next, expose the Pod with a NodePort Service: -->
+接着，使用 NodePort Service 公开 Pod
```
Comment (tengqm) [inline content/zh/docs/tutorials/security/seccomp.md:664]:
```suggestion
接着，使用 NodePort Service 公开 Pod：
```

File: content/zh/docs/tutorials/security/seccomp.md [hunk 17]
```diff
@@ -0,0 +1,783 @@
...
+如果你已经启用了 `SeccompDefault` [特性门控](/zh/docs/reference/command-line-tools-reference/feature-gates/)，
+只要没有指定其他 seccomp 配置文件，那么 Pod 就会使用 `SeccompDefault` 的 seccomp 配置文件。
+否则，默认值为 `Unconfined`。
+{{< /note >}}
+
+<!-- 
+Here's a manifest for a Pod that requests the `RuntimeDefault` seccomp profile
+for all its containers:
+-->
+这是一个 Pod 的清单，它为其所有容器请求 `RuntimeDefault` seccomp 配置文件：
```
Comment (tengqm) [inline content/zh/docs/tutorials/security/seccomp.md:744]:
```suggestion
这是一个 Pod 的清单，它要求其所有容器使用 `RuntimeDefault` seccomp 配置文件：
```

File: content/zh/docs/tutorials/security/seccomp.md [hunk 18]
```diff
@@ -0,0 +1,783 @@
...
+
+<!-- The Pod should be showing as having started successfully: -->
+此 Pod 应该显示为成功启动：
+```
+NAME        READY   STATUS    RESTARTS   AGE
+default-pod 1/1     Running   0          20s
+```
+
+<!-- Finally, now that you saw that work OK, clean up: -->
+最后，既然你看到可以运行，请清理：
```
Comment (tengqm) [inline content/zh/docs/tutorials/security/seccomp.md:766]:
```suggestion
最后，你看到一切正常之后，请清理：
```

**Comment by jihoon-seo** [comment]
Please notice that:
- Renamed in EN: [en/docs/tutorials/clusters/seccomp.md → en/docs/tutorials/security/seccomp.md](https://github.com/kubernetes/website/commit/ec94e69c4ef0ea63692c312038f16c8fc2e1b6b7#diff-0f73ee3a4f9254371d7563c378b1ff89057d7fb140b2a2698d3cc33635352805)
- ZH version exists: https://kubernetes.io/zh/docs/tutorials/clusters/seccomp/

**Comment by howieyuen** [comment]
> Please notice that:
> 
> * Moved in EN: [en/docs/tutorials/clusters/seccomp.md → en/docs/tutorials/security/seccomp.md](https://github.com/kubernetes/website/commit/ec94e69c4ef0ea63692c312038f16c8fc2e1b6b7#diff-0f73ee3a4f9254371d7563c378b1ff89057d7fb140b2a2698d3cc33635352805)
> * ZH version exists: https://kubernetes.io/zh/docs/tutorials/clusters/seccomp/

thanks, already fix the pr content

**Pull request merged**